esp32/modules/easyrtc.py

```diff
@@ -18,15 +21,15 @@ def string(date=False, time=True):
     hourstr = str(hour)
     if (hour<10):
       hourstr = "0"+hourstr
-    minstr = str(min)
-    if (min<10):
+    minstr = str(minute)
```

Comment:
I understand things like this enhance readability, but you're also extending the string matching time and ram usage